### ISSUE TITLE
Windows "high contrast mode"affecting Brave

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#14501]
## Description 

If Windows's high contrast mode is enabled, Brave seems to incorrectly take on some of the Windows theming, making it display incorrectly.  This was  previously not an issue and only started after the update to v1.21.73. 

It appears that some users are seeing this happen in [Chrome as well](https://community.brave.com/t/option-to-not-match-braves-theme-with-windows/212626/2?u=mattches), but majority of users are claiming that it's only Brave and that other browsers (Chrome, Edge) are not showing this same behavior.
cc @kjozwiak @rebron 

## Steps to Reproduce

   1. Enable high contrast mode in Windows 10 Settings
   2. Open and browse Brave
   3. 

## Actual result:

![image](https://user-images.githubusercontent.com/3299098/110156524-ed536300-7d9b-11eb-8301-f69d57fa01bb.png)
![image](https://user-images.githubusercontent.com/3299098/110156555-f7756180-7d9b-11eb-9b4c-1ecff69a2735.png)
![image](https://user-images.githubusercontent.com/3299098/110156594-052ae700-7d9c-11eb-81c7-6166dee613a8.png)
![image](https://user-images.githubusercontent.com/3299098/110156598-09570480-7d9c-11eb-8709-dae3183b0cc4.png)

## Reproduces how often: 

I personally was not able to reproduce, but it  seems to be a pretty simple state to get into.

## Brave version (brave://version info)

Version 1.21.73 Chromium: 89.0.4389.72 (Official Build) (64-bit)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? N/A
- Can you reproduce this issue with the nightly channel? N/A

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? Yes/No

## Miscellaneous Information:

Community threads:
https://community.brave.com/t/home-page-and-webpages-not-rendering-correctly/212312
https://community.brave.com/t/option-to-not-match-braves-theme-with-windows/212626/2
https://community.brave.com/t/windows-high-contrast-setting-affecting-brave/212625

**Comment by Brave-Matt** [comment]
Additional Community report +1 
https://community.brave.com/t/brave-blocking-webpage-colors/214192/7

**Comment by Un1qu3-PL** [comment]
There you go.
1. Type **chromium://flags** or **brave://flags** to enter the experimental settings.
2. Search for **Forced Colors**
3. Disable and relaunch Brave.
![image](https://user-images.githubusercontent.com/80531413/110955473-ef906100-8349-11eb-9115-bac9374eae98.png)
![image](https://user-images.githubusercontent.com/80531413/110955488-f5864200-8349-11eb-9e57-82b61e2eefdc.png)

**Comment by mentalisttraceur** [comment]
I think this default behavior is a feature, not a bug - high contrast mode is first and foremost an accessibility aid. If I'm flipping on high-contrast mode to help me see, I expect and want it to terraform websites too.

Ideally though, I think they should expose an option in the settings (in the actual user-friendly *settings*, not in the technical `chrome://flags` backrooms) to control forced colors mode (match OS / always on (customize colors) / always off), just like Firefox does.

Even more ideally, the first time Brave sees you using high contrast mode in the OS, it would *ask* if you want websites rendered that way, and point you to where the toggle is in the settings.

**Comment by mentalisttraceur** [comment]
By the way, some users like myself also use "high contrast" mode (with custom gentler lower-contrast dark colors) as a much more reliable+consistent forced dark mode across the whole system - which is still an accessibility thing I'd argue, just for a different need than violently high contrast.

Sometimes I end up having to still customize CSS for webpages (because too many people making websites don't give a damn about accessibility, so some websites just become unusable in forced colors mode), but it's more reliably not painful-to-the-eyes to use forced colors than all the mainstream forced-dark-mode attempts out there.

**Comment by NemoArmitage** [comment]
Brave 1.80.113 (Offizieller Build) (64-Bit)
Chromium: 138.0.7204.49

High Contrast mode affects the browser again.

Froced colors flag has disappeared.
Is that how it should be?
How can I fix this without disabling High Contrast mode in Windows 10?

![Image](https://github.com/user-attachments/assets/9083db6a-9509-473b-95bb-60f7d07f516c)

**Comment by evanrgardner** [comment]
+1 High contrast mode preventing user from seeing buttons and certain settings 

<img width="466" height="621" alt="Image" src="https://github.com/user-attachments/assets/01872699-6120-4db9-a847-cb3272218e47" />
